kuzzleio/documentation-V2
# Refactor new file structure for SDK documentation

## What does this PR do?

Update files structure, snippets and sections managements for a better way to write SDK documentation
```
| - index
    | - create
        | - snippets
             | - usage.cpp
             | - usage.go
             | - usage.test.yml
         | - index.md
         | - create.cpp.md
         | - create.js.md
```
### How should this be manually tested?

go here : https://deploy-preview-14--doc-v2.netlify.com/sdk-reference/index/create/
and check everythings is ok.


### Other changes

 * fix [#9](https://github.com/kuzzleio/documentation-V2/issues/9)
 * change hilight.js to prism.js for a better code display in front
 * little enhancement of CSS
